packages/cashier/src/components/cashier-onboarding/cashier-onboarding.jsx

```diff
@@ -37,8 +37,10 @@ const CashierOnboarding = ({
 }) => {
     const history = useHistory();
     const is_crypto = !!currency && isCryptocurrency(currency);
-    const has_crypto_account = accounts_list.some(x => x.is_crypto);
-    const has_fiat_account = accounts_list.some(x => !x.is_crypto);
+    const has_crypto_account = Object.values(accounts).some(acc_settings => isCryptocurrency(acc_settings.currency));
+    const has_fiat_account = Object.values(accounts).some(
+        acc_settings => !acc_settings.is_virtual && !isCryptocurrency(acc_settings.currency)
+    );
```

Comment:
@george-usynin-binary Better to memorize them, Otherwise they will run in each re-render.